ui/admin/app/components/model-providers/ModelProviderModels.tsx

```diff
@@ -84,18 +84,19 @@ export function ModelProvidersModels({ modelProvider }: ModelsConfigureProps) {
 
     function getColumns(): ColumnDef<Model, string>[] {
         return [
-            columnHelper.accessor((model) => model.id, {
-                id: "id",
+            columnHelper.accessor((model) => model.name, {
+                id: "name",
                 header: "Model",
             }),
             columnHelper.display({
                 id: "usage",
                 header: "Usage",
-                cell: ({ row }) => (
-                    <Badge variant="outline">
-                        {getModelUsageLabel(row.original.usage)}
-                    </Badge>
-                ),
+                cell: ({ row }) =>
+                    getModelUsageLabel(row.original.usage) ? (
+                        <Badge variant="outline">
+                            {getModelUsageLabel(row.original.usage)}
+                        </Badge>
+                    ) : null,
```

Comment:
should this be sortable on the usage label?

```suggestion
            columnHelper.accessor((model) => getModelUsageLabel(model.usage) || "", {
                id: "usage",
                header: "Usage",
                cell: ({ row, getValue /* or something */ }) =>
                    getValue() ? (
                        <Badge variant="outline">
                            {getValue()}
                        </Badge>
                    ) : null,
```

Reply:
I realize we haven't built out sorting into our data tables, but I think we should prepare for that likelihood :)

Reply:
Yeah especially with a large list of models, I see that happening and a search to filter as well!